src/DirectoryCleaner.php

```diff
@@ -30,6 +33,14 @@ public function setDirectory(string $directory)
         return $this;
     }
 
+    public function setExtensions(string $extensions)
+    {
+        if ($extensions && $extensions != "*"){
```

Comment:
Please use strict comparisons.

Use an early return if you don't want to do anything on `*`